lib/internal/test_runner/mock.js

```diff
@@ -202,6 +202,60 @@ class MockTracker {
     return mock;
   }
 
+  getter(
+    object,
+    methodName,
+    implementation = kDefaultFunction,
+    options = kEmptyObject
+  ) {
+    if (implementation !== null && typeof implementation === 'object') {
+      options = implementation;
+      implementation = kDefaultFunction;
+    }
+
+    const { getter = true } = options;
+
+    validateBoolean(getter, 'options.getter');
+
+    if (!getter) {
+      throw new ERR_INVALID_ARG_VALUE(
+        'options.getter', getter, "cannot set to 'false'"
```

Comment:
Same comment for setter:

```suggestion
        'options.getter', getter, "cannot be false"
```